src/lib/Microsoft.Health.Logger/Telemetry/IomtTelemetryLogger.cs

```diff
@@ -54,16 +52,30 @@ public void LogMetricWithDimensions(Common.Telemetry.Metric metric, double metri
             metric.LogMetric(_telemetryClient, metricValue);
         }
 
+        private void LogExceptionWithProperties(Exception ex)
+        {
+            EnsureArg.IsNotNull(ex, nameof(ex));
+            ex.LogException(_telemetryClient);
```

Comment:
nit: just my personal opinion - usually log methods are done on a logger/telemetry client instance, so on first glance this pattern looks different where we are calling it on an exception and passing in the client. 
Nothing wrong with it though, maybe I have just been used to seeing the other way more often :)

Reply:
Same here, but I'll keep this pattern since we also have it for metrics (c.f. `LogMetricWithDimensions`).

Or, considering your other PR comment, maybe the metric/exception extension classes can be combined into 1 telemetry client extension class. What do you think?

Reply:
I think that makes sense to have a common telemetry extension class.

Reply:
@pallar-ms - I made updates based on these comments. Would you like to take another look at the new commits?